### ISSUE TITLE
Errors with duckplyr

### ISSUE THREAD

**Issue by krlmlr** [josesamos/starschemar#7]
When checking starschemar with a version of dplyr that uses duckdb under the hood, I'm seeing errors that I'm having a hard time understanding.

- dplyr version: https://github.com/krlmlr/dplyr/tree/f-revdep-duckplyr
- results: https://github.com/krlmlr/dplyr/blob/eeda3997c65efa8b954e8c46826684c29caf911a/revdep/problems.md#newly-broken-14

The error message is: 

```
! `x` must be a vector, not a <tbl_df/tbl/data.frame/dimension_table> object.
```

It seems odd that `"dimension_table"` comes last in the list of classes, even after `"data.frame"` . Is this intended? The typical way would be to put it first, as in `class = c("dimension_table", "tbl_df", "tbl", "data.frame")` .

Thank you for your support.

I also wonder if and how your package could interoperate with dm: https://github.com/cynkra/dm/issues/2212.

**Comment by josesamos** [comment]
The order of the classes had been corrected in the code but remained in the data included in the package. I have modified all the datasets according to the current code so that the dimension_table or fact_table classes are now the first in the list. 

I think that would be corrected with that and can it could interoperate with dm. It's already updated on GitHub, I'm going to send it to CRAN. 

I'm sorry for any possible inconvenience.
